routes/api.php

```diff
@@ -39,6 +39,7 @@ static function (): void {
         Route::resource('users', UserController::class)->except('create', 'edit');
         Route::post('users/{id}/resume', [ResumeController::class, 'store']);
         Route::get('user', [UserController::class, 'showSelf']);
+        Route::post('user/override/auto', [UserController::class, 'applySelfOverride']);
```

Comment:
I don't think `auto` makes sense here, maybe `self` instead.
```suggestion
        Route::post('user/override/self', [UserController::class, 'applySelfOverride']);
```